src/main/io/ledstrip.c

```diff
@@ -543,10 +549,33 @@ static void applyLedFixedLayers(void)
             break;
 
         case LED_FUNCTION_BATTERY:
+        case LED_FUNCTION_BATTERY_BAR:
             color = HSV(RED);
             hOffset += MAX(scaleRange(calculateBatteryPercentageRemaining(), 0, 100, -30, 120), 0);
             break;
 
+#ifdef USE_GPS
+        case LED_FUNCTION_GPS_BAR:
```

Comment:
ledBarIdx as for LED_FUNCTION_BATTERY_BAR. But bar mode makes sense only when some sats are available